content/articles/2017-08-08-contributing.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Contributing Code to the Tidyverse
+slug: contributing
+date: 2018-08-08
+categories: [Other]
+---
+
+Contributing code to open source projects can be intimidating. These projects
+are often widely used and have well known maintainers. Contributing code and
+having it accepted seems an almost insurmountable task.
+
+However if you follow a few simple strategies you can have your code accepted
+into even the most popular projects in the [tidyverse](https://tidyverse.org).
+
+### Don't Contribute Code at all ###
```

Comment:
Can you please use sentence case?

I think you could probably drop "at all" too